README.md

```diff
@@ -81,32 +85,30 @@ You can add up to 10 optional templates for:
 You do this by creating files under the folders `templates/article_pages`, `templates/category_pages` or `templates/section_pages`.
 Learn more [here](https://support.zendesk.com/hc/en-us/articles/360001948367).
 
-## Styles
-The styles that Theming Center needs to use in the theme are in the `style.css` file in the root folder.
+## Stylesheet and JavaScript
 
-The styles for the theme are split using Sass partials, all the partials are under [styles/](/blob/master/styles/), they are all included in the "main" file [index.scss](/blob/master/styles/index.scss) and then compiled to CSS.
-If you wish to use SASS you can go to the [using SASS section](#using-sass)
+We use Rollup to compile the JS and CSS files that are used in the theme - `style.css` and `script.js`. Do not edit these directly as they'll be regenerated during release.
 
-## Assets
-The Copenhagen theme doesn't have any assets, but you can add assets to your theme by placing them in the `assets` folder.
-
-# Using SASS
-In order to use SASS for development, you just need to compile it into the CSS that Zendesk Guide understands.
-Note: Zendesk App Tools [theme preview](#publishing-your-theme) currently does not support live SASS compilation.
+To get started:
 
-## Requirements
-
-- Install Ruby, we use `sassc` gem to compile our `.scss` files. You can see how to install Ruby [here](https://www.ruby-lang.org/en/documentation/installation/).
-- Install `sassc` gem. You can run:
 ```console
-  gem install sassc:1.12.1
+$ yarn install --offline
```

Comment:
This PR does not add the yarn packages to the repo. Should we remove the `--offline` option here?

```suggestion
$ yarn install
```